nominees/heroes-bot.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Heroes Bot",
+  "description": "A Telegram bot that allows frontline healthcare workers to log symptoms they are experiencing, get counseling, best practice information, logistical assistance and provide creative ways to distress.",
+  "website": "https://github.com/africa-covid-19-response-toolkit/heroes-bot",
```

Comment:
@318anushka kindly update the website field to point to a websiteURL rather than Github repo. Thanks.